node/standalone/src/chain_spec.rs

```diff
@@ -151,18 +127,6 @@ fn testnet_genesis(
 	endowed_accounts: Vec<AccountId>,
 	_enable_println: bool,
 ) -> GenesisConfig {
-	let gerald_evm_account_id = H160::from_str("6be02d1d3665660d22ff9624b7be0551ee1ac91b").unwrap();
-	let mut evm_accounts = BTreeMap::new();
-	evm_accounts.insert(
-		gerald_evm_account_id,
-		evm::GenesisAccount {
-			nonce: 0.into(),
-			balance: U256::from(123456_123_000_000_000_000_000u128),
-			storage: BTreeMap::new(),
-			code: vec![],
-		},
-	);
-	log::info!("Adding balance for {}", gerald_evm_account_id);
```

Comment:
For our own knowledge, and to help users, we should figure out the consequences of initializing balances in both pallets (pallet-evm, and pallet-balances), and either document or disable it.

My guess is that whichever pallet comes later in `construct_runtime!` will write on top of whatever came first. That would mean that accounts could be initialized in either place, but if an account was initialized in _both_ places, the later one would overwrite the former.

Reply:
Yes, that is what I observed and my expectation looking at the code, but we could add a warning if we detect it.